commands/ls_test.go

```diff
@@ -43,6 +43,11 @@ func TestParseFiltersAll(t *testing.T) {
 	assert.Equal(t, actual, FilterOptions{SwarmName: []string{"foo"}, DriverName: []string{"bar"}, State: []string{"Stopped"}, Name: []string{"dev"}})
 }
 
+func TestParseFiltersAllCase(t *testing.T) {
+	actual, _ := parseFilters([]string{"sWarM=foo", "DrIver=bar", "StaTe=Stopped", "NAMe=dev"})
```

Comment:
The error should not be swallowed, the test should contain an `assert` check that it was not `nil`.  Can you please make a followup PR to fix this?